tests/unit/models/UserNotificationsTest.php

```
@@ -60,8 +90,29 @@ public function testPatch(): void
+        $this->UserNotifications->patch(Action::Update, array());
+    }
+
     public function testDestroy(): void
     {
         $this->assertTrue($this->UserNotifications->destroy());
     }
+
+    public function testDestroyNotifOfAnotherUser(): void
+    {
+        $Notif = new SelfIsValidated($this->Users);
+        $id = $Notif->create();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$id'. (undefined)

(UnusedLocalVariable)